component/core/src/main/java/org/exoplatform/social/core/identity/model/Profile.java

```diff
@@ -171,7 +171,8 @@
     EXPERIENCES,
     AVATAR,
     ABOUT_ME,
-    BANNER;
+    BANNER,
+    LAST_LOGIN_TIME;
```

Comment:
Should we rename to use TECHNICAL ? so that is could be used for lastLoginTime, external and so on